src/MAUI/Maui.Samples/Samples/Data/ManageFeatures/ManageFeatures.xaml

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage x:Class="ArcGIS.Samples.ManageFeatures.ManageFeatures"
+             xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:esriUI="clr-namespace:Esri.ArcGISRuntime.Maui;assembly=Esri.ArcGISRuntime.Maui">
+    <Grid>
+        <esriUI:MapView x:Name="MyMapView" />
+        <Border Style="{StaticResource EsriSampleControlPanel}"
+                VerticalOptions="Start">
+            <StackLayout Padding="5"
+                         Spacing="5">
+                <Label Text="Select an option." />
```

Comment:
Maybe some label like `Select CRUD operation.`

Reply:
I agree with Hamish. I am in favor of a `:` instead of `.` and utilizing bold text it comes to picker instructions. But that's me being nitpicky.

Reply:
I was thinking "Select an edit operation:" as I don't intend to maintain the "CRUD" term